core/src/main.rs

```diff
@@ -143,12 +140,18 @@ fn main_loop_iteration(args: &Args) -> Result<()> {
             Ok(live_epoch_state) => PoolState::LiveEpoch(live_epoch_state),
             Err(_) => PoolState::NeedsBootstrap,
         };
-        let datapoint_source: Box<dyn DataPointSource> =
-            if let Some(external_script_name) = &args.datapoint_script_name {
-                Box::new(ExternalScript::new(external_script_name.clone()))
-            } else {
-                Box::new(NanoErgUsd)
-            };
+        let datapoint_source: Box<dyn DataPointSource> = match &*op.data_point_source {
+            "NanoErgUsd" => Box::new(NanoErgUsd),
+            "NanoAdaUsd" => Box::new(NanoAdaUsd),
+            "CustomScript" => {
```

Comment:
Do you think it's necessary for `data_point_source` in the config file to have "CustomScript" value? I mean if `data_point_source_custom_script` is set than we can ignore `data_point_source` altogether, right? Sort of override.

Reply:
Yes of course, it's done.